### DISCUSSION TITLE
Which Infinispan Quarkus Extension should be used

### DISCUSSION THREAD

**Discussion by mschorsch** [quarkusio/quarkus#42016] [Q&A] [answered]
Hello 👋, i've found different Quarkus extensions for Infinispan:

- https://github.com/infinispan/infinispan/tree/main/quarkus
- https://github.com/quarkusio/quarkus/tree/main/extensions

Which one should be used? The extension in the Quarkus repo seems not to support embedded infinispan, is this planned for future versions?

[a comment by an automated account is not shown]

**Comment by karesti** [comment, marked as answer]
@mschorsch if you are using quarkus-cache, you want to move from caffeine to Infinispan, and use @CacheResult annotations and impact as less as possible your code, use quarkus-infinispan-cache. Simply use the Infinispan client extension if you want to use Infinispan. The embedded use case is not on the roadmap now.
